Exercise_D and E/assets/style/index.css

```diff
@@ -1,4 +1,89 @@
-/* 
-This css sheet is linked to practice.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+* {
+    margin: 0;
+    flex-direction: column;
+    max-width: 100%;
+    max-height: 100%;
+    position: relative;
+  }
+  
+  body {
+    background-color: rgb(184, 218, 155);;
+  }
+  
+  nav {
+    background-color: white;
+  }
+  
+  .stall-container {
+      position: relative;
+  }
+  
+  .stall-container p {
+    font-size: 60px;
+    color: white;
+    display: flex;
+    position: absolute;
+    top: 55%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-weight: bold;
+    
+  }
+  
+  
+  .stall-back {
+  
+    background-color: black;
+  }
+  
+  #flowers {
+    width: 100%;
+    opacity: 0.5;
+  }
+  
+  button {
```

Comment:
You can do add a little more css to make the button reflect the image that we are asking you to copy